### PR TITLE
volume-options: store req.Options in volinfo

### LINKED ISSUE #366: GET /v1/volumes/<volname> doesn't display volume options set through volume options API
While I was trying to set some volume options through volume options API, the command succeeded. However get volume API returned blank options field.

### PR DESCRIPTION
**Pull request by atinmu**
Fixes #366
Signed-off-by: Atin Mukherjee <amukherj@redhat.com>

### CODE REVIEW

**Comment by kshlm** [comment]
add to whitelist

File: commands/volumes/volume-option.go [hunk 1]
```diff
@@ -58,6 +58,10 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		restutils.SendHTTPError(w, http.StatusBadRequest, fmt.Sprintf("invalid option specified: %s", err.Error()))
 		return
 	}
+	// Update the options into volinfo
+	for k, v := range req.Options {
+		volinfo.Options[k] = v
+	}
```
Comment (kshlm) [inline commands/volumes/volume-option.go]: There is a loop that does exactly this on line 106, before `txn.Do()` is called. Why doesn't it work there?

File: commands/volumes/volume-option.go [hunk 2]
```diff
@@ -58,6 +58,10 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		restutils.SendHTTPError(w, http.StatusBadRequest, fmt.Sprintf("invalid option specified: %s", err.Error()))
 		return
 	}
+	// Update the options into volinfo
```
Comment (aravindavk) [inline commands/volumes/volume-option.go]: Already this code is available in line 106. Looks like some other issue while storing in volinfo

Comment (atinmu) [reply commands/volumes/volume-option.go]: Correct, I overlooked this code. Let me figure it out where the problem is.

**Comment by prashanthpai** [comment]
PR #374  fixes the issue.

**Pull request closed without merging**